blog/2024-02-08-year-of-agama.mdx

```diff
@@ -0,0 +1,106 @@
+---
+date: 2024-02-08 12:00:00 +00:00
+title: The Year of Agama
+description: Take a look to the Agama roadmap for 2024
+tags:
+- project
+- systems-management
+- YaST
+- agama
+- programming
+---
+
+At the end of 2023 we announced Agama 7 stating that version is the first prototype we could consider
+to be (quoting ourselves) "functional enough", covering areas such as localization, network
+configuration, storage setup, authentication basis and some software selection. Now it's time to go
+deeper into every area... and we have a plan for that.
+
+{/* truncate */}
+
+## The Agama Roadmap for 2024 {#roadmap}
```

Comment:
I know you are just importing, but I wonder if we should adapt the capitalization. See https://github.com/agama-project/agama/pull/1633.